lib/ansible/modules/system/systemd.py

```diff
@@ -307,7 +307,7 @@ def main():
     # check service data, cannot error out on rc as it changes across versions, assume not found
     (rc, out, err) = module.run_command("%s show '%s'" % (systemctl, unit))
 
-    if out.find('ignoring request') != -1:
+    if err.find('ignoring request') != -1:
```

Comment:
Fair. I've attempted to avoid false positives by making sure '=' is not in the output in the updated version.